firehose/model.py

```diff
@@ -233,7 +235,7 @@ def from_json(cls, jsonobj):
         raise TypeError('unknown type: %r' % jsonobj['type'])
 
 class Issue(Result):
-    attrs = [Attribute('cwe', int, nullable=True),
+    attrs = [Attribute('cwe', list, nullable=True),
              Attribute('testid', _string_type, nullable=True),
```

Comment:
"cwe" should probably be replaced by "externalids" or somesuch, for external IDs.  (Not sure of the name).  It would be a list of instances of some base class, with CWE being a subclass.